mesos_slave/assets/configuration/spec.yaml

```
@@ -0,0 +1,33 @@
+name: Mesos Slave
+
+files:
+  - name: mesos_slave.yaml
+    options:
+      - template: init_config
+        options:
+          - template: init_config/default
+          - template: init_config/http
```

```suggestion
          - template: init_config/http
          - template: init_config/default
```
Same here for init_config